packages/cdk/lib/transcription-service.ts

```diff
@@ -166,6 +166,9 @@ export class TranscriptionService extends GuStack {
 		// basic placeholder commands
 		userData.addCommands(
 			[
+				// this should be done in AMIgo but for now need to do linux postinstall here https://docs.docker.com/engine/install/linux-postinstall/
+				`usermod -aG docker ubuntu`,
+				`newgrp docker`,
```

Comment:
Now that https://github.com/guardian/amigo/pull/1406 is in we can remove this change